app/core/language.js

```diff
@@ -9,64 +9,88 @@ import mainFR from '../../locales/fr/translation.json';
 import parameterFR from '../../locales/fr/parameter.json';
 import mainDE from '../../locales/de/translation.json';
 import parameterDE from '../../locales/de/parameter.json';
+import mainIT from '../../locales/it/translation.json';
 
-// Update moment with the right language, for date display
-i18n.on('languageChanged', lng => {
-  moment.locale(lng);
-  if (self.localStorage) {
-    self.localStorage.lang = lng;
-  }
-});
+const crowdinActive = typeof _jipt === 'object';
 
-let defaultLanguage = getLocale();
+let language = getLocale();
```

Comment:
would it be more efficient to have this one as an _else_ condition of the below?

Reply:
no